docs/DeveloperGuide.md

```diff
@@ -298,15 +298,23 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 ### Non-Functional Requirements
 
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+2.  Should be able to hold up to 1000 tasks without a noticeable sluggishness in performance for typical usage.
+3.  The report is expected to be generated and formatted within 2 seconds.
```

Comment:
as of v1.2 there is no feature that requires the generation of reports

Reply:
Should I just delete it or mark it as coming soon?